src/Template/Element/Form/included.twig

```diff
@@ -0,0 +1,21 @@
+{% if included %}
+{% set imageattributes = { 0: 'url' } %}
```

Comment:
cycle through `included` array (see above) checking that `type` is `streams`
or use only first element